EKrut-Server/src/ekrut/server/db/ReportDAO.java

```diff
@@ -434,6 +450,37 @@ public ArrayList<String> getAllCustomersOrders(LocalDateTime date, String locati
 				}
 			}
 	}
+	// Add document
+	public ArrayList<LocalDateTime> getAllCustomersOrdersByDate(LocalDateTime date, String location) {
+		
+		PreparedStatement ps1 = con.getPreparedStatement("SELECT username FROM orders WHERE"
+				+ " EXTRACT(MONTH FROM date) = EXTRACT(MONTH FROM ?) AND"
+				+ " EXTRACT(YEAR FROM date) = EXTRACT(YEAR FROM ?) AND location = ?");
+		try {
+			ps1.setObject(1, date, MysqlType.DATETIME);
+			ps1.setString(2, location);
```

Comment:
You have 3 parameters in the prepared statement but you're only setting 2 of them.